DataFormats/CaloTowers/interface/CaloTower.h

```diff
@@ -198,6 +207,12 @@ class CaloTower : public reco::LeafCandidate {
    GlobalPoint emPosition_;
    GlobalPoint hadPosition_;
 
+  //hcal limits in CT continuous ieta (from CaloTowerTopology)
+  int firstHBRing_, lastHBRing_;
```

Comment:
Actually, I like the enum idea, it seems like the most straightforward change. Would it be acceptable just to reuse the [HcalSubdetector](https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/DataFormats/HcalDetId/interface/HcalSubdetector.h) enums, or is this sort of dependency not desirable?